include/deal.II/grid/grid_tools.h

```diff
@@ -1019,6 +1019,15 @@ namespace GridTools
    * identify the cells in the output Triangulation and corresponding cells in
    * the input list.
    *
+   * The function copies the location of vertices of cells from the cells of the
+   * source triangulation to the triangulation that is built from the list of
+   * patch cells.  This adds support for triangulations which have been
+   * perturbed or smoothed in some manner after the refinement or in short for
+   * any triangulation that does not follow from the standard refinement
+   * strategy employed in deal.ii of creating new vertices on midpoints of
+   * faces or edges. (Note: there is still not support for isoperimetric
```

Comment:
there -> There

The word is also "isoparametric". "isoperimetric" refers to something different. But I'd just remove the parenthetic remark. The documentation describes what the function does, not what it might do if extended.